package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "babel-preset-es2015": "^6.24.0",
     "body-parser": "^1.18.2",
+    "dotenv": "^5.0.1",
     "eslint": "^3.17.1",
```

Comment:
it's a good practice to add eslint into the dev dependencies. This helps keep packages that are only used in development out of  your production build.